src/components/Card.jsx

```diff
@@ -0,0 +1,74 @@
+import styled from 'styled-components';
+import { StyledCard } from './styles/Card.styled';
+import PropTypes from 'prop-types';
+
+const Thumbnail = styled.img`
+  height: 150px;
+  object-fit: cover;
+  border-radius: 10px;
+  margin-bottom: 10px;
+`;
+
+const CardTitle = styled.h2`
+  font-size: 20px;
+  font-weight: 600;
+  margin: auto;
+  color: #18191a;
+`;
+
+const CardTitleContainer = styled.div`
+  display: -webkit-box;
+  -webkit-line-clamp: 2;
+  -webkit-box-orient: vertical;
+  overflow: hidden;
+  padding: 0px 15px;
+`;
+
+const CardDescription = styled.p`
+  font-size: 16px;
+  text-overflow: ellipsis;
+  color: #18191a;
+`;
+
+const CardDescriptionContainer = styled.div`
+  display: -webkit-box;
+  -webkit-line-clamp: 4;
+  -webkit-box-orient: vertical;
+  overflow: hidden;
+  margin-top: 2%;
+  padding: 0 15px;
+`;
```

Comment:
Best if you move this styled components to be on its own. As the ones you have below